README.md

```diff
@@ -1,21 +1,134 @@
-# bs-node-serialport
-A binding class to enable using https://www.npmjs.com/package/serialport on BrightSign players
+# Basic Reference Implementation - JavaScript Serial Application
 
-Refer to the included example. 
+This reference implementation is an example Node app built to demonstrate sending Serial Data over the BrightSign's 3.5mm Serial connector to be received on the BrightSign's USB-A port.
 
-In order to use the serialport library on a BrightSign player:
+It introduces an approach to write HTML, Javascript, and Node.js for the BrightSign player. This is intended to be developed on your local machine, run and tested on a BrightSign unit. BrightAuthor:connected is not required for this application. 
 
-1. Create a SerialPort instance 
-2. Create a BrightSign binding instance (see https://serialport.io/docs/api-bindings) 
-3. Create a parser instance, (see https://serialport.io/docs/api-parsers-overview)
-4. Assign the the BrightSign binding to the SerialPort
-5. For reading, pipe the SerialPort to the parser
-6. For writing, use the "write" method of the SerialPort. See the example in index.js, where the string "abc" is written
+This reference currently assumes USB-A is sending to USB-C serial protocol. It may be the case the Player's log output is over the 3.5mm Serial Port or SSH/Telnet. 
 
-This example includes a custom BrightScript (autorun.brs) that creates the Node.js application, but you can also create an HTML 5 widget in BrightAuthor. Remember to check the box "Enable node.js".
+## Getting Started
+---
+After cloning the repo and changing to it's directory, do the following.
 
+```bash
+npm i
+npm run build:dev
+```
+
+Deploy the bundled application to the player through the DWS APIs. See 
+[Enabling development tools on the Player ](#enable-player-development-tools) to enable the DWS, and [Deploying Code Remotely](#deploy-code) to deploy the code. 
```

Comment:
The links on this line do not work.

Reply:
I think github doesnt render markdown anchor links well in the raw view - could you switch your files edit view to the "Display rich diff" view (the button to right of the <> button at the top of your window). Do the links render better?